src/behavior-considered-undefined.md

```diff
@@ -16,9 +16,9 @@ undefined behavior, it is *unsound*.
 
 ***Warning:*** The following list is not exhaustive. There is no formal model of
 Rust's semantics for what is and is not allowed in unsafe code, so there may be
-more behavior considered unsafe. The following list is just what we know for
-sure is undefined behavior. Please read the [Rustonomicon] before writing unsafe
-code.
+more behavior considered unsafe. We also reserve the right to make some of the
+behavior in that list defined in the future.
```

Comment:
```suggestion
behavior in that list defined in the future, or conversely, to guarantee that it will always be undefined.
```

Reply:
I am confused, this now says "we reserve the right to [...] guarantee that it will always be undefined"?

Reply:
I think Josh is trying to capture the point that Niko raised below: We may want to _promise_ that certain operations will always be UB, so that unsafe code can rely on that.

Reply:
I don't understand the point of saying "in the future we may want to promise that something will always be UB". On its own, how is that statement useful for anyone? I would understand the point of marking particular items in the UB list as "this will definitely always be UB", but we don't currently have any such items, or do we?

Reply:
@RalfJung The wording could absolutely change. The wording currently in the PR says that we could *define* it in the future. The point Niko made, which I wanted to capture, was that we could also commit to *not* defining something (or for that matter commit to not defining it in particular ways), which is something that some crates may need us to do.

Reply:
Hm, I still don't understand whom this comment is serving/helping. People will be asking for us to commit to something like that no matter what we write here. But sure I can try to put in a phrase along those lines. I just worry it will cause a lot more confusion than it will help people. It would certainly confuse the heck out of me without a multi-paragraph explanation.

Reply:
I tried my hand at it, but honestly I don't like it. I think the text would be semantically equivalent but less confusing without that new sentence.